sale_loyalty_order_line_link/migrations/16.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_renamed_fields = [
+    (
+        "sale.order.line",
+        "sale_order_line",
+        "coupon_program_id",
+        "loyalty_program_id",
+    ),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_fields(env, _renamed_fields)
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE sale_order_line
+        ADD COLUMN IF NOT EXISTS reward_id INTEGER;
```

Comment:
Don't put semicolons in the statements
```suggestion
        ADD COLUMN IF NOT EXISTS reward_id INTEGER
```